tests/meta-mender-toradex-nxp-ci/recipes-bsp/u-boot/files/colibri-imx7/fw_env.config

```diff
@@ -0,0 +1,10 @@
+# Configuration file for fw_(printenv/setenv) utility.
+# Up to two entries are valid, in this case the redundant
+# environment sector is assumed present.
+# Notice, that the "Number of sectors" is not required on NOR and SPI-dataflash.
+# Futhermore, if the Flash sector size is ommitted, this value is assumed to
+# be the same as the Environment size, which is valid for NOR and SPI-dataflash
+
+# MTD device name       Device offset   Env. size       Flash sector size       Number of sectors
+/dev/mtd3               0x00000000      0x00020000      0x20000                 4
+/dev/mtd3               0x00020000      0x00020000      0x20000                 4
```

Comment:
@mirzak can you double check something for me?

 I'm looking at vf support, the BOOTENV_SIZE is 0x20000, ENV_OFFSET is 0x180000 (I suppose you've set MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET to the same value when building locally), ENV_RANGE is 0x80000 (4 * ENV_SIZE). VF eraseblock is 128kB, so I'm guessing you've set MENDER_PARTITION_ALIGNMENT_KB to 128 as well.

The generated `config_mender_defines.h` will define:
```
#define MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_1 0x180000 /* CONFIG_ENV_OFFSET */
#define MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_2 0x1a0000 /* ??? */
```
Then the 'generic mender bootcode' patch will set:
```
+#define CONFIG_ENV_OFFSET        MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_1
+#define CONFIG_ENV_OFFSET_REDUND MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_2
```
The offset of redundant environment will be 0x1a0000 (0x20000 wrt. /dev/mtd2 or u-boot-env start), while it should be 0x200000 (0x80000 wrt. /dev/mtd2, as defined in `fw_env.config`)